<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainForm</class>
  <widget class="QWidget" name="MainForm">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1100</width>
        <height>700</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>SmartDesk Manager</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
        <number>0</number>
      </property>
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>

      <!-- SIDEBAR -->
      <item>
        <widget class="QFrame" name="sidebar">
          <property name="minimumSize">
            <size>
              <width>260</width>
              <height>0</height>
            </size>
          </property>
          <property name="maximumSize">
            <size>
              <width>260</width>
              <height>16777215</height>
            </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_sidebar">
            <property name="spacing">
              <number>0</number>
            </property>
            <property name="leftMargin">
              <number>0</number>
            </property>
            <property name="rightMargin">
              <number>0</number>
            </property>
            <property name="topMargin">
              <number>0</number>
            </property>

            <item>
              <widget class="QLabel" name="label_logo">
                <property name="text">
                  <string>SmartDesk</string>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
              </widget>
            </item>

            <item>
              <widget class="QPushButton" name="btn_nav_dashboard">
                <property name="text">
                  <string>üìä Dashboard</string>
                </property>
                <property name="checkable">
                  <bool>true</bool>
                </property>
                <property name="checked">
                  <bool>true</bool>
                </property>
                <property name="autoExclusive">
                  <bool>true</bool>
                </property>
                <property name="class" stdset="0">
                  <string>nav-btn</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="btn_nav_desktops">
                <property name="text">
                  <string>üñ•Ô∏è Desktops</string>
                </property>
                <property name="checkable">
                  <bool>true</bool>
                </property>
                <property name="autoExclusive">
                  <bool>true</bool>
                </property>
                <property name="class" stdset="0">
                  <string>nav-btn</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="btn_nav_hotkeys">
                <property name="text">
                  <string>‚å®Ô∏è Hotkeys</string>
                </property>
                <property name="checkable">
                  <bool>true</bool>
                </property>
                <property name="autoExclusive">
                  <bool>true</bool>
                </property>
                <property name="class" stdset="0">
                  <string>nav-btn</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="btn_nav_tray">
                <property name="text">
                  <string>üîß Tray Icon</string>
                </property>
                <property name="checkable">
                  <bool>true</bool>
                </property>
                <property name="autoExclusive">
                  <bool>true</bool>
                </property>
                <property name="class" stdset="0">
                  <string>nav-btn</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="btn_nav_settings">
                <property name="text">
                  <string>‚öôÔ∏è Einstellungen</string>
                </property>
                <property name="checkable">
                  <bool>true</bool>
                </property>
                <property name="autoExclusive">
                  <bool>true</bool>
                </property>
                <property name="class" stdset="0">
                  <string>nav-btn</string>
                </property>
              </widget>
            </item>

            <item>
              <spacer name="verticalSpacer">
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>

      <!-- CONTENT -->
      <item>
        <widget class="QStackedWidget" name="stackedWidget">

          <!-- PAGE 1: DASHBOARD -->
          <widget class="QWidget" name="page_dashboard">
            <layout class="QVBoxLayout" name="verticalLayout_dash">
              <property name="leftMargin">
                <number>40</number>
              </property>
              <property name="topMargin">
                <number>40</number>
              </property>
              <property name="rightMargin">
                <number>40</number>
              </property>

              <item>
                <widget class="QLabel" name="label_dash_title">
                  <property name="text">
                    <string>Dashboard</string>
                  </property>
                  <property name="class" stdset="0">
                    <string>page-title</string>
                  </property>
                </widget>
              </item>

              <!-- Stats Grid -->
              <item>
                <layout class="QHBoxLayout" name="horizontalLayout_stats">
                  <item>
                    <widget class="QGroupBox" name="group_stats">
                      <property name="title">
                        <string>Statistiken</string>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_stats">
                        <item row="0" column="0">
                          <widget class="QLabel" name="label_stat_total">
                            <property name="text">
                              <string>Total: -</string>
                            </property>
                          </widget>
                        </item>
                        <item row="0" column="1">
                          <widget class="QLabel" name="label_stat_active">
                            <property name="text">
                              <string>Aktiv: -</string>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </item>
                </layout>
              </item>

              <item>
                <widget class="QGroupBox" name="group_status">
                  <property name="title">
                    <string>System Status</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_status">
                    <item>
                      <widget class="QTextEdit" name="text_status_log">
                        <property name="readOnly">
                          <bool>true</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QPushButton" name="btn_refresh_dashboard">
                        <property name="text">
                          <string>Status aktualisieren</string>
                        </property>
                        <property name="class" stdset="0">
                          <string>action-btn</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </item>

              <item>
                <spacer name="verticalSpacer_dash">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>

        </widget>
      </item>
    </layout>
  </widget>
</ui>
